concrete-table-inheritance/src/main/java/com/iluwatar/concretetableinheritance/App.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Permission is hereby granted, free of charge, to any person obtaining a copy.
+ */
+
+package com.iluwatar.concretetableinheritance;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.ibatis.session.SqlSession;
+
+/**
+ * The App class to run the application.
+ */
+@Slf4j
+public final class App {
+
+  /**
+   * constructor.
+   */
+  private App() { }
+
+  /**
+   * This is main entrance.
+   *
+   * @param args input args.
+   */
+  public static void main(String[] args) {
+
+    final SqlSession sqlSession = Mybatis3Utils.getCurrentSqlSession();
```

Comment:
I guess we should ensure that this gets closed, right? You could use try-with-resources.